content/en/docs/ops/common-problems/injection/index.md

```diff
@@ -148,7 +148,7 @@ Verify the `caBundle` in the `mutatingwebhookconfiguration` matches the
 {{< text bash >}}
 $ kubectl get mutatingwebhookconfiguration istio-sidecar-injector -o yaml -o jsonpath='{.webhooks[0].clientConfig.caBundle}' | md5sum
 4b95d2ba22ce8971c7c92084da31faf0  -
-$ kubectl -n istio-system get secret istiod-service-account-token -o jsonpath='{.data.root-cert\.pem}' | md5sum
+$ kubectl -n istio-system get secret istio-ca-secret -o jsonpath='{.data.root-cert\.pem}' | md5sum
```

Comment:
This is empty in my out-of-box `istioctl install`

Reply:
Yes, `root-cert.pem` is empty. The correct field seems to be `ca-cert.pem`. I fixed it in a new commit.